mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -549,12 +549,13 @@ struct EraseSelfCopyOnBuffers : OpRewritePattern<CopyOp> {
   using OpRewritePattern<CopyOp>::OpRewritePattern;
```

Comment:
The `OnBuffers` part of the name of this pattern can be dropped.

Reply:
done